streams/th.m3u

```diff
@@ -14,11 +12,9 @@ http://news1.live14.com/stream/news1.m3u8
 #EXTINF:-1 tvg-id="Channel5.th",Channel 5 (1080p)
 https://639bc5877c5fe.streamlock.net/tv5hdlive/tv5hdlive/playlist.m3u8
 #EXTINF:-1 tvg-id="Channel7.th",Channel 7 (1080p)
-http://streaming-hwc.ch7.com/livech7hd/HD_1080p.m3u8
+http://live-cdn-hwc.ch7.com/livech7hd/HD_1080p.m3u8
```

Comment:
Tried https://live-cdn-hwc.ch7.com/livech7hd/HD.m3u8 returns 403
Please confirm if it works for you, might be geo-blocked

Reply:
Works if you're in Thailand only. Geo.

Reply:
![snapshot](https://github.com/iptv-org/iptv/assets/30985701/e7ea5792-5f81-4c15-9fbe-37aa87b18038)